Source/WebCore/dom/ScriptExecutionContext.cpp

```diff
@@ -249,11 +250,11 @@ void ScriptExecutionContext::dispatchMessagePortEvents()
     auto completionHandlers = std::exchange(m_processMessageWithMessagePortsSoonHandlers, Vector<CompletionHandler<void()>> { });
 
     // Make a frozen copy of the ports so we can iterate while new ones might be added or destroyed.
-    for (auto* messagePort : copyToVector(m_messagePorts)) {
+    for (RefPtr messagePort : copyToVector(m_messagePorts)) {
```

Comment:
I'd use `copyToVectorOf<RefPtr<MessagePort>>` here because otherwise we're still iterating over a vector of raw pointers, which isn't great for safety.